luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/decide.htm

```diff
@@ -7,39 +7,32 @@ <h3>Freifunk-Assistent</h3>
 
   <div class="info alert-message">
     <p class="success">
-<%= translate("Als naechstes kommen die Einstellungen für das Freifunk-Netzwerk. \
-Halte die IPs bereit, die du ueber den Wizard von config.berlin.freifunk.net \
-bekommen hast.") %>
+<%= translate("Next we set the options for the Freifunk network. Keep the IPs ready that you got from the config.berlin.freifunk.net wizard.") %>
```

Comment:
Since we are changing things anyway, would it be ok to have the text

"Have the IPs ready that you received from the config.berlin.freifunk.net wizard"

Reply:
And some like breaks would be nice.